model/gcp/registry.yaml

```diff
@@ -195,3 +195,28 @@ groups:
         brief: >
           The hostname of a GCE instance. This is the full value of the default or [custom hostname](https://cloud.google.com/compute/docs/instances/custom-hostname-vm).
         examples: ['my-host1234.example.com', 'sample-vm.us-west1-b.c.my-project.internal']
+
+  - id: registry.gcp.gen_ai
+    type: attribute_group
+    display_name: Generative AI Attributes
+    brief: >
+      This documents defines attributes for Google generative AI systems.
+    stability: development
+    attributes:
+      - id: gcp.gen_ai.request
+        stability: development
+        type: template[string[]]
```

Comment:
do you want value to be an array of strings or a single string?

I.e. `"foo.bar.baz": "5"` or `"foo.bar.baz": ["5"]`  ?

Reply:
I actually want the value types to be `any`, but unfortunately this doesn't look to be supported.

Filed: https://github.com/open-telemetry/weaver/issues/705

Reply:
No longer applicable. Marking resolved.